python/cuml/fil/fil.pyx

```diff
@@ -95,7 +95,7 @@ cdef class TreeliteModel():
         return out
 
     @staticmethod
-    def from_filename(filename, model_type="xgboost"):
+    def from_filename(filename, model_type):
```

Comment:
I think we should keep the default model type, but maybe you have another approah in mind

Reply:
ohh i think that got lost when i pulled and had merge conflicts, i will add that in again, sorry